esp/templates/program/modules/admincore/deadlines.html

```diff
@@ -180,13 +180,13 @@ <h3>Edit Deadlines</h3>
             {% for p in perm.includes %}
                 {% if perm.open_now or p.bit_open %}
                 <li>{% if p.overridden %}<span class="override">
-                    <a href="#perm_form_{{ p.overridden_by.id }}">
```

Comment:
Hm, why this change?

Reply:
I thought moving it down to the next line (so that it's only linked if it's actually overridden) made more sense so people only click if it's relevant

Reply:
Oh, I see now.  Ok.